test/src/main/java/org/apache/accumulo/test/ScanServerConcurrentTabletScanIT.java

```diff
@@ -65,25 +69,94 @@ public static void start() throws Exception {
     ScanServerConcurrentTabletScanITConfiguration c =
         new ScanServerConcurrentTabletScanITConfiguration();
     SharedMiniClusterBase.startMiniClusterWithConfig(c);
-    SharedMiniClusterBase.getCluster().getClusterControl().start(ServerType.SCAN_SERVER,
-        "localhost");
+  }
+
+  @AfterClass
+  public static void stop() throws Exception {
+    SharedMiniClusterBase.stopMiniCluster();
+  }
+
+  private void startScanServer(boolean cacheEnabled)
+      throws IOException, KeeperException, InterruptedException {
 
     String zooRoot = getCluster().getServerContext().getZooKeeperRoot();
     ZooReaderWriter zrw = getCluster().getServerContext().getZooReaderWriter();
     String scanServerRoot = zooRoot + Constants.ZSSERVERS;
 
+    SharedMiniClusterBase.getCluster().getClusterControl().stop(ServerType.SCAN_SERVER);
+
+    Map<String,String> overrides = new HashMap<>();
+    overrides.put(Property.SSERV_CACHED_TABLET_METADATA_EXPIRATION.getKey(),
+        cacheEnabled ? "5m" : "0m");
```

Comment:
Maybe set the cache timeout really high so it never happens even if the test it running on a really slow machine.

```suggestion
        cacheEnabled ? "300m" : "0m");
```